### ISSUE TITLE
std/node does not work in Worker

### ISSUE THREAD

**Issue by Jack-Works** [denoland/deno_std#1626]
It requires access to `window` which is not available in the worker.

```
error: Uncaught (in worker "") ReferenceError: window is not defined
    window.addEventListener("unload", () => {
    ^
    at new Process (https://deno.land/std@0.114.0/node/process.ts:344:5)
```

**Comment by lucacasonato** [comment]
`window` should be replaced with `self`.

**Comment by Jack-Works** [comment]
Or globalThis

**Comment by bartlomieju** [comment]
This was already fixed in #1543, upgrade to `std@0.115.1`
